pkg/descheduler/descheduler.go

```diff
@@ -318,6 +319,8 @@ func RunDeschedulerStrategies(ctx context.Context, rs *options.DeschedulerServer
 			eventRecorder,
 		)
 
+		parentCtx, _, parentCloser := tracing.StartSpan(ctx, "descheduler_parent", "deschedule")
```

Comment:
can this just re-assign to the existing `ctx` instead of creating a new variable?